packages/mds-transactions-service/repository/index.ts

```diff
@@ -58,15 +60,27 @@ class TransactionReadWriteRepository extends ReadWriteRepository {
       return {}
     }
     function who(): { provider_id?: UUID } {
-      if (provider_id) {
-        return { provider_id }
-      }
-      return {}
+      return provider_id ? { provider_id } : {}
     }
     try {
       const connection = await connect('ro')
-      const entities = await connection.getRepository(TransactionEntity).find({ where: { ...who(), ...when() } })
-      return entities.map(TransactionEntityToDomain.mapper())
+      const queryBuilder = connection
+        .getRepository(TransactionEntity)
+        .createQueryBuilder('transactionentity') // yuk!
+        .where({ ...who(), ...when() })
+      const query: PagingQuery = {
+        limit: 10,
```

Comment:
I'm kinda waiting for @cjlynch12 to express an opinion, but i will add `limit`.

Reply:
`limit` will have to have some cap so that we can't be dos'd.

Reply:
@karcass 
Sorry missed this tag.

We've been using 10000 as the default/max limit for abt event endpoints, as a point of reference. 
Our high water mark for # of monthly transactions is in the 1.2mil-ish range. The ui should support up to 1 billing cycle (month) of transactions at a time.

Fine to leave this as 10 if it's over-ride-able, otherwise we'll need to bump this higher so we don't need to spam requests.